extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/KafkaStreamsTopologyManager.java

```diff
@@ -168,6 +159,42 @@ private void waitForTopicsToBeCreated(Collection<String> topicsToAwait, String b
         }
     }
 
+    public Set<String> getMissingTopics(Collection<String> topicsToCheck, String bootstrapServersConfig)
+            throws InterruptedException {
+        final Map<String, Object> config = getAdminClientConfig(bootstrapServersConfig);
```

Comment:
Shouldn't this config be created only once?

Reply:
You're right. It's even simpler.